doc/radosgw/s3/bucketops.rst

```diff
@@ -537,7 +537,8 @@ Parameters are XML encoded in the body of the request, in the following format:
 | ``Topic``                     | String    | Topic ARN. Topic must be created beforehand                                          | Yes      |
 +-------------------------------+-----------+--------------------------------------------------------------------------------------+----------+
 | ``Event``                     | String    | List of supported events see: `S3 Notification Compatibility`_.  Multiple ``Event``  | No       |
-|                               |           | entities can be used. If omitted, all events are handled                             |          |
+|                               |           | entities can be used. If omitted, all "Created" and "Removed" events are handled.    |          |
+|                               |           | "Lifecycle" and "Synced" event types must be specified explicitly.                   |          |
```

Comment:
these are just ordinary create/deletes, so yes

Reply:
@cbodley: I assumed so. However: if not, we can just close this (and close it in Quincy too), and this comment will remain as an explanation to posterity of why this commit wasn't ingested into the release branches at this time.

Reply:
oh, you meant lifecycle; so no, and so far we've not planned to put those into reef

Reply:
@zdover23 of the future: PR#54499 is in main/latest as of 15 Nov 2023. The state of its backports is evident above in this PR.

Reply:
but I don't think Yuval's PR necessarily intended to convey that ObjectLifecycle events would be taken for reef--if we do want them, that's possible, tho

Reply:
@mattbenjamin I see these "lifecycle notifications" commits:
2a465f806ae70fc82259dd26713f79bbcc2b8d41
06eb0df290042bea6e866c4edeea82bfa79833d0
7b74603513981c622fb3d25d8cd13f13a4c4dcb0
35a4eb4410394a0014648dda7df92642f3b536d3
in "quincy".

and these "sync notifications" commits:
cc5354e1c4b6743b2b525dfd81db526726bfec2c
b303ae1f117e19bbc7b8f6bed4eb21fa50733865
in "reef".

Reply:
@mattbenjamin: Should this go into the Squid docs?

Reply:
@zdover23 @mattbenjamin what is blocking this PR?

Reply:
@yuvalif, Nothing is blocking this PR from being merged except for its lack of an approving review. I'll merge it the moment I get one. I would approve it myself and merge it, but since I raised it, I can't.